src/penn_chime/models.py

```diff
@@ -259,31 +267,56 @@ def sir(
 
 def gen_sir(
     s: float, i: float, r: float, gamma: float, i_day: int, policies: Sequence[Tuple[float, int]]
-) -> Generator[Tuple[int, float, float, float], None, None]:
+):
     """Simulate SIR model forward in time yielding tuples.
```

Comment:
No longer yielding tuples, but returning dataframe

Reply:
Not actually a dataframe, just an 'append' only dictionary of named arrays. The dataframe construction was basically all of the execution cost!

Reply:
Yes, of course the dataframe is expensive!